.vscode/settings.json

```diff
@@ -11,5 +11,8 @@
   "typescript.preferences.quoteStyle": "single",
   "prettier.singleQuote": true,
   "cSpell.words": ["adal", "uninitialize"],
-  "jest.jestCommandLine": "pnpm test -- --"
+  "jest.jestCommandLine": "pnpm test -- --",
+  "[typescript]": {
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
What is this for?

Reply:
I think this is just the personal `.vscode` config that got accidentally committed and should probably be removed